### PR TITLE
Added a sub-project about training the model on the CurveLanes dataset

### PR DESCRIPTION
**Pull request by pandamax**
hi @koyeongmin，First of all thanks for your wonderful job. Meanwhile I added a sub-project about training the model on the curvelanes dataset based on your project. it contrains the script for how to convert the CurveLanes dataset to the tusimple-like format  and  a example for how to convert the trained model to onnx and run inference  using onnx model. Hope it can help your experiment and expand the application of the model. Also hope to get your feedback and suggestions. Thanks again!

### CODE REVIEW

**Comment by gneworld** [comment]
> hi @koyeongmin，First of all thanks for your wonderful job. Meanwhile I added a sub-project about training the model on the curvelanes dataset based on your project. it contrains the script for how to convert the CurveLanes dataset to the tusimple-like format and a example for how to convert the trained model to onnx and run inference using onnx model. Hope it can help your experiment and expand the application of the model. Also hope to get your feedback and suggestions. Thanks again!

@pandamax 

hi pandamax, thanks for your great job, I have a try with this code on curvelanes dataset, find that couldn't reach your training result, my training result and your pretrained model like this
image

so how should I do to get loss 1.1001 after only 32 epochs?  thanks very much

**Comment by pandamax** [comment]
@gneworld Have you converted the CurveLanes into TuSimple-like format? And `lane_agent.save_model(int(step/1000), loss_p)`**so 32 is not the epoch num ** I have not change any hyperparameters compared with TuSimple sub-project when training the model,maybe it should make some change on hyperarameters setting base on different  dataset.And It seems that the your training loss value is normal.

**Comment by gneworld** [comment]
@pandamax  I made a misunderstanding about the meaning of "30", it actually means 30000 steps. I have converted  CurveLanes into TuSimple-like format with your supplied curvelanes2tusimples.py and any hyperparameters is the same as yours. In addition, I find that loss value of the first 1000 step is unstable, sometimes it is ~10 or ~9， ~8，like this 

Training loop
epoch : 0 loss_p: tensor(8.9110)
epoch : 0 loss_p: tensor(4.6231)
epoch : 0 loss_p: tensor(3.7302)
......


Training loop
epoch : 0 loss_p: tensor(10.8118)
epoch : 0 loss_p: tensor(5.9708)
epoch : 0 loss_p: tensor(3.8624)
......

I don't know if it is right?

**Comment by koyeongmin** [comment]
Thank you very much for your contribution!!!

**Comment by gneworld** [comment]
@pandamax Do you train the model from scratch or with pretrained model(which model)?
